rust/libnewsboat/src/utils.rs

```diff
@@ -102,6 +103,13 @@ pub fn resolve_tilde(path: String) -> String {
     return file_path;
 }
 
+pub fn resolve_relative(reference: &Path, path: &Path) -> PathBuf {
+	if path.is_relative() {
+		return reference.parent().unwrap().join(path);
```

Comment:
Please either handle both possible results of `parent()`, or write a code comment explaining why it can't possibly fail.